/**:
  ros__parameters:
    device_name: /dev/ttyACM_c
    baud_rate: 115200
    flow_control: none
    parity: none
    stop_bits: "1"
